hw3/Author.rb

```diff
@@ -0,0 +1,36 @@
+class Author
+    attr_reader :name, :biography, :yearOfBirth
+    
+    def initialize(name, yearOfBirth, biography='')
+        unless validName?(name) 
```

Comment:
raise 'Not valid name'  unless validName?(name)
raise 'Not valid yearOfBirth' unless validYearOfBirth?(yearOfBirth)